src/pages/AzurePreview.js

```diff
@@ -1,89 +1,235 @@
-      textTransform: 'none'
+    onSubmit: e => {
+      e.preventDefault()
+      onSubmit()
     }
-  }
+  }, [
+    fields.map(({ key, label }) => {
+      const inputId = `azure-preview-interest-${key}`
+      return div({ key, style: { width: 250 } }, [
+        h(FormLabel, { htmlFor: inputId, required: true }, [label]),
+        h(ValidatedInput, {
+          inputProps: {
+            id: inputId,
+            value: formValue[key],
+            onChange: value => {
+              setFieldsTouched(v => ({ ...v, [key]: true }))
+              onChange({ ...formValue, [key]: value })
+            },
+          },
+          error: fieldsTouched[key] && !formValue[key] ? `${label} is required` : undefined,
+        }),
+      ])
+    }),
+
+    input({ type: 'submit', value: 'submit', style: { display: 'none' } })
+  ])
+}
 
-  const supportEmail = 'preview@terra.bio'
-  const supportSubject = 'Terra on Microsoft Azure Preview Environment'
+const formId = '1FAIpQLSegf8c7LxlVOS8BLNUrpqkiB7l8L7c135ntdgaBSV2kdrqSAQ'
 
-  const dismiss = () => {
-    azurePreviewStore.set(true)
+const formInputMap = {
+  firstName: 'entry.1708226507',
+  lastName: 'entry.677313431',
+  title: 'entry.1500649388',
+  organization: 'entry.1020670185',
+  contactEmail: 'entry.1125156163',
+  terraEmail: 'entry.1938956483',
+}
+
+const AzurePreviewForNonPreviewUser = () => {
+  const [hasSubmittedForm, setHasSubmittedForm] = useState(() => getLocalPref(submittedPreviewFormPrefKey) || false)
+  useEffect(() => {
+    setLocalPref(submittedPreviewFormPrefKey, hasSubmittedForm)
+  }, [hasSubmittedForm])
+
+  const [busy, setBusy] = useState(false)
+
+  const [userInfo, setUserInfo] = useState(() => {
+    const user = getUser()
+
+    // If the user's name contains only one space, guess that it contains
+    // their first and last name and auto populate those inputs.
+    // Otherwise, leave them blank.
+    const nameParts = (user.name || '').trim().split(/\s/)
+    const [firstName, lastName] = nameParts.length === 2 ? nameParts : ['', '']
+
+    return {
+      firstName,
+      lastName,
+      title: '',
+      organization: '',
+      contactEmail: user.email,
+      terraEmail: user.email,
+    }
+  })
+
+  const submitEnabled = Object.values(userInfo).every(Boolean) && !busy
+
+  const submitForm = useCallback(async () => {
+    setBusy(true)
+    try {
+      const formInput = Object.entries(formInputMap)
+        .reduce((acc, [userInfoKey, formFieldId]) => ({ ...acc, [formFieldId]: userInfo[userInfoKey] }), {})
+
+      await Ajax().Surveys.submitForm(formId, formInput)
+      setHasSubmittedForm(true)
+    } catch (error) {
+      reportError('Error submitting information', error)
+    } finally {
+      setBusy(false)
+    }
+  }, [userInfo])
+
+  if (hasSubmittedForm) {
+    return h(Fragment, [
+      p({ style: styles.paragraph }, [
+        'Thank you for your interest in using Terra on Azure. We will be in touch with your access information soon.'
```

Comment:
MSFT had previously asked to say `Terra on Microsoft Azure` instead of `Terra on Azure`